recipes-extended/libcobalt/libcobalt_git.bb

```diff
@@ -53,45 +58,33 @@ def get_cobalt_data_path(d):
         return "${WPEFRAMEWORK_DATA_PATH}/Cobalt"
 COBALT_DATA = "${@get_cobalt_data_path(d)}"
 
-do_configure() {
-    export PATH="$PATH:${S}/depot_tools"
+do_compile() {
     export COBALT_EXECUTABLE_TYPE="shared_library"
     export COBALT_HAS_OCDM="${@bb.utils.contains('DISTRO_FEATURES', 'opencdm', 1, 0, d)}"
     export COBALT_HAS_PROVISION="${@bb.utils.contains('DISTRO_FEATURES', 'provisioning', 1, 0, d)}"
     export COBALT_HAS_WAYLANDSINK="${@bb.utils.contains('DISTRO_FEATURES', 'weston', 1, 0, d)}"
+    export COBALT_VIDEO_OVERLAY=${COBALT_VIDEO_OVERLAY}
 
     export COBALT_STAGING_DIR="${STAGING_DIR_HOST}/"
+    export OPENSSL_NO_ASM=1
     export COBALT_TOOLCHAIN_PREFIX="${STAGING_DIR_NATIVE}${bindir}/${TARGET_SYS}/${TARGET_PREFIX}"
+    export PYTHONPATH="${PYTHONPATH}:${S}"
     export COBALT_DATA_PATH="${COBALT_DATA}/data"
 
-    export COBALT_INSTALL_DIR="${D}"
-    "${S}/src/cobalt/build/gyp_cobalt" -C "${COBALT_BUILD_TYPE}" "${COBALT_PLATFORM_NAME}"
-}
-
-do_compile() {
-    export PATH="$PATH:${S}/depot_tools"
-    "${STAGING_BINDIR_NATIVE}/ninja" -C "${S}/src/out/${COBALT_PLATFORM_NAME}_${COBALT_BUILD_TYPE}" cobalt_deploy
+    cd "${S}" && "${STAGING_BINDIR_NATIVE}"/gn gen out/wpe --script-executable=python3 --args='target_platform="${COBALT_PLATFORM}" build_type="${COBALT_BUILD_TYPE}" target_cpu="arm" is_clang=false sb_install_content_subdir="${COBALT_DATA}/data" is_video_overlay=${COBALT_VIDEO_OVERLAY}'
+    "${STAGING_BINDIR_NATIVE}/ninja" -C "${S}"/out/wpe cobalt_install
 }
 
 do_install() {
-    export PATH="$PATH:${S}/../depot_tools"
-    if [ -f "${STAGING_DIR_TARGET}/usr/lib/libcobalt.so" ]
-    then
-        rm -rf "${STAGING_DIR_TARGET}/usr/lib/libcobalt.so"
-    fi
     install -d "${D}${libdir}"
-    install -m 0755 "${S}/src/out/${COBALT_PLATFORM_NAME}_${COBALT_BUILD_TYPE}/lib/libcobalt.so" "${D}${libdir}"
+    install -m 0755 "${S}/out/wpe/install/lib/libcobalt.so" "${D}${libdir}"
+    install -d "${D}${includedir}"/starboard
+    cp -prf "${S}"/starboard/*.h "${D}${includedir}"/starboard/
+    install -d "${D}${includedir}"/third_party/starboard/wpe/shared
```

Comment:
Please double quote the whole expression here and below

Reply:
done